### PR TITLE
Customizer Widgets: Fix inserter button size and animation

### PR DESCRIPTION
**Pull request by t-hamano**
Related to #67440

## What?

In the Customizer widget, do two things:

- Change the button to 32px square
- Add an animation when the inserter is opened

## Why?

The inserter button was previously 24px in size:

image

Due to the influence of #67440, it seems that only the button width has changed from 24px to 32px.

## How?

We could have changed it back to its original 24px size, but since the visual size of all the other buttons in this toolbar is 32px, I've changed the inserter button to be 32px square as well.

As for animations, the same styles are applied as in the Editor and Widget Editor:

https://github.com/WordPress/gutenberg/blob/b1d943f027532a9e83b2739930d218c67f7f930d/packages/editor/src/components/document-tools/style.scss#L18-L25

## Testing Instructions

- Activate Twenty Twenty-One
- Appearance > Customize > Widgets

## Screenshots or screencast 

| State | Before | After |
|--------|--------|--------|
| Default | image  | image |
| Inserter Opened | image | image |

### CODE REVIEW

File: packages/customize-widgets/src/components/header/style.scss [hunk 1]
```diff
@@ -33,16 +33,25 @@
...
+		svg {
+			transition: transform cubic-bezier(0.165, 0.84, 0.44, 1) 0.2s;
+			@include reduce-motion("transition");
+		}
+
 		&.is-pressed {
 			background: $gray-900;
+
+			svg {
+				transform: rotate(45deg);
```
Comment (tyxla) [inline packages/customize-widgets/src/components/header/style.scss:53]: Neat, thanks for adding the animation ❤️

**Review by tyxla — APPROVED** [review]
LGTM, thank you @t-hamano 👍 

Screenshot 2024-12-12 at 16 34 31
Screenshot 2024-12-12 at 16 34 19

**Review by jameskoster — APPROVED** [review]
Looks good. Thank you ❤️

**Comment by t-hamano** [comment]
Thanks for the review!

**Pull request merged**